### PR TITLE
Lambda.part1.exercise.exercise1(Fedulov)

### PR DESCRIPTION
**Pull request by merceneryinbox**

### CODE REVIEW

File: src/test/java/lambda/part1/exercise/Exercise1.java [hunk 1]
```diff
@@ -1,84 +1,121 @@
...
+		}
+		// TODO использовать Arrays.sort
+		
+		
+		assertArrayEquals(new Person[]{
+				new Person("Иван", "Мельников", 20),
+				new Person("Николай", "Зимов", 30),
+				new Person("Алексей", "Доренко", 40),
+				new Person("Артем", "Зимов", 45)}, persons);
+		Arrays.parallelSort(persons);
```
Comment (Boochok) [inline src/test/java/lambda/part1/exercise/Exercise1.java]: Why do you sort persons after the assertion?

File: src/test/java/lambda/part1/exercise/Exercise1.java [hunk 2]
```diff
@@ -61,53 +64,58 @@ public int compare(Person o1, Person o2) {
...
 		
 		
-		return new Person[]{new Person("Иван", "Мельников", 20),
-							new Person("Алексей", "Доренко", 40),
-							new Person("Николай", "Зимов", 30), new Person("Артем", "Зимов", 45)};
+		return new Person[]{
+				new Person("Иван", "Мельников", 20),
+				new Person("Николай", "Зимов", 30),
+				new Person("Алексей", "Доренко", 40),
+				new Person("Артем", "Зимов", 45)};
```
Comment (Boochok) [inline src/test/java/lambda/part1/exercise/Exercise1.java]: in my point of view you should sort array of persons in the methods, not change the task

File: src/test/java/lambda/part1/exercise/Exercise1.java [hunk 3]
```diff
@@ -1,84 +1,113 @@
...
+	
+	@Test
+	public void sortPersonsByLastNameThenFirstNameUsingArraysSortAnonymousComparator() {
+		Person[] persons = getPersons();
+		
+		Arrays.sort(persons, new Comparator<Person>() {
+			@Override
+			public int compare(Person o1, Person o2) {
+				return o1.getLastName().compareTo(o2.getLastName()) == 0 ? 0 :
+					   o1.getFirstName().compareTo(o2.getFirstName());
```
[the remark this answers is not in the document]
Comment (merceneryinbox) [reply src/test/java/lambda/part1/exercise/Exercise1.java]: +

File: src/test/java/lambda/part1/exercise/Exercise1.java (hunk 1 above)
Comment (merceneryinbox) [reply src/test/java/lambda/part1/exercise/Exercise1.java]: +

File: src/test/java/lambda/part1/exercise/Exercise1.java (hunk 2 above)
Comment (merceneryinbox) [reply src/test/java/lambda/part1/exercise/Exercise1.java]: +

**Pull request closed without merging**